src/modules/hero/hero.component.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { DebugElement } from '@angular/core';
```

Comment:
Separate libraries with new lines so that they have logical groupings (angular, blackbaud, internal).